### PR TITLE
Add question to capture all logits for target tokens

### PR DESCRIPTION
**Pull request by Dylan102938**

### CODE REVIEW

File: llmcompare/runner/tokens.py [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+        tokens = encoding.encode(tok)
+        assert len(tokens) == 1, f'Expected single token for "{tok}", got {tokens}'
+        token_ids.append(tokens[0])
+
+    return token_ids
+
+
+def get_tokens(token_ids: list[int], encoding_scheme: str | None = None) -> list[str]:
+    encoding = get_encoding(encoding_scheme)
+    return [encoding.decode([int(tok)]) for tok in token_ids]
```
Comment (johny-b) [inline llmcompare/runner/tokens.py:29]: I'd remove `int` here as this function already expects ints. Like, let it fail if someone uses the function with wrong input. No strong opinions.

File: llmcompare/runner/runner.py [hunk 2]
```diff
@@ -84,10 +86,50 @@ def single_token_probs(
         result = dict(sorted(result.items(), key=lambda x: x[1], reverse=True))
         return result
 
+    def specific_token_logits(
+        self,
+        messages: list[dict],
+        tokens: list[int] | list[str],
```
Comment (johny-b) [inline llmcompare/runner/runner.py:92]: I think it's better to have two arguments for this.
Main reason: if someone passes
`["123"]`
when they intended to pass
`[123]`
this might be very hard to debug. This happens to me sometimes!

File: llmcompare/question/question.py [hunk 3]
```diff
@@ -581,3 +588,29 @@ def get_runner_input(self) -> list[dict]:
...
             el["top_logprobs"] = self.top_logprobs
         return runner_input
+
+
+class Logits(Question):
+    _runner_sampling_func_name = "specific_token_logits"
+
+    def __init__(
+        self,
+        tokens: list[int] | list[str],
```
Comment (johny-b) [inline llmcompare/question/question.py:598]: See this comment: https://github.com/johny-b/llmcompare/pull/9#discussion_r2191791262
